src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -46,6 +50,7 @@ export class HuiDialogEditCard extends LitElement {
             params.path[0]
           ].cards![params.path[1]])
         : undefined;
+    this._newCard = params.path.length === 2 ? true : false;
```

Comment:
I think that we should make this a property, generated on the fly. That way there is a single source of truth.

```ts
private get _newCard() {
  return this._params!.path.length === 2;
}
```

Reply:
That's not even necessary. The value never really changes, so it can just be set in the constructor.